translations/binaries/slps_strings_city_names.json

```
@@ -0,0 +1,816 @@
+            "offset": 1733048,
+            "string": "The Hidden Temple[END]",
+            "translation": "The Hidden Temple[END]",
+            "original_bytes_length": 18,
+            "available_bytes_size": 23
+        },
+        {
+            "offset": 1733072,
+            "string": "隠された神殿[END]",
+            "original_bytes_length": 14,
+            "available_bytes_size": 15,
+            "translation": "숨겨진 신전[END]"
+        },
+        {
+            "offset": 1733088,
+            "string": "フィッツガルドの北部にある村。[LINE]\nほとんどの村民が農業と牧畜を営んでいる。[END]",
+            "original_bytes_length": 72,
+            "available_bytes_size": 72,
+            "translation": "피츠갈드의 북부에 위치한 마을.[LINE]\n거의 모든 마을 주민이 농업과 목축을 하고 있다.[END]"
+        },
+        {
+            "offset": 1733160,
+            "string": "Leane[END]",
+            "translation": "Leane[END]",
+            "original_bytes_length": 6,
+            "available_bytes_size": 7
+        },
+        {
+            "offset": 1733168,
+            "string": "リーネの村[END]",
+            "original_bytes_length": 11,
+            "available_bytes_size": 15,
+            "translation": "리네의 마을[END]"
+        },
+        {
+            "offset": 1733184,
+            "string": "セインガルドの北東海域の島に造られた[LINE]\nオベロン社の工場。[LINE]\n現在は閉鎖されている。[END]",
+            "original_bytes_length": 79,
+            "available_bytes_size": 79,
+            "translation": "세인갈드의 북동 해역에 세워진[LINE]\n오베론사의 공장.[LINE]\n현재는 폐쇄되어 있다.[END]"
+        },
+        {
+            "offset": 1733264,
+            "string": "Oberon's Hidden Factory[END]",
+            "translation": "Oberon's Hidden Factory[END]",
+            "original_bytes_length": 24,
+            "available_bytes_size": 24
+        },
+        {
+            "offset": 1733288,
+            "string": "オベロン秘密工場[END]",
+            "original_bytes_length": 18,
+            "available_bytes_size": 23,
+            "translation": "오베론 비밀 공장[END]"
+        },
+        {
+            "offset": 1733312,
+            "string": "セインガルドの東部にある街。[LINE]\n郊外の街らしく、ゆっくりとした時間が[LINE]\n流れている。[END]",
+            "original_bytes_length": 79,
+            "available_bytes_size": 79,
+            "translation": "세인갈드의 동부에 위치한 도시.[LINE]\n교외의 도시답게, 느긋한 시간이[LINE]\n흘러가고 있다.[END]"
+        },
+        {
+            "offset": 1733392,
+            "string": "Cresta[END]",
+            "translation": "Cresta[END]",
+            "original_bytes_length": 7,
+            "available_bytes_size": 7
+        },
+        {
+            "offset": 1733400,
+            "string": "クレスタの街[END]",
+            "original_bytes_length": 13,
+            "available_bytes_size": 15,
+            "translation": "크레스타의 도시[END]"
+        },
+        {
+            "offset": 1733416,
+            "string": "セインガルドの首都。[LINE]\n整備された城下町は、人々で賑わっている。[END]",
+            "original_bytes_length": 62,
+            "available_bytes_size": 63,
+            "translation": "세인갈드의 수도.[LINE]\n정비된 성곽 마을은 사람들로 북적이고 있다.[END]"
```

```suggestion
            "translation": "세인가르드의 수도.[LINE]\n성을 중심으로 정비된 도시는 사람들로 북적이고 있다.[END]"
```

원문인 城下町가 옛날 일본 성하고 관련된 거라 마땅한 단어가 애매해서 그냥 의미 그대로 풀어서 썼습니다

---

단순히 성(성곽)으로 둘러싸인 도시를 의미하는 것이라면 '성곽 도시'도 적절한 표현일 것 같아요. 옛날 일본 성의 특이한 형태를 일컫는 용어인가보죠? 실제 적용할 때 글자 수가 모자라지 않는다면, 적어주신 표현을 우선할게요.